src/snyk/common/languageServer/middleware.ts

```diff
@@ -1,5 +1,7 @@
-import { IConfiguration } from '../configuration/configuration';
-import {
+import { CLI_INTEGRATION_NAME } from '../../cli/contants/integration';
+import { Configuration, IConfiguration } from '../../common/configuration/configuration';
+import { User } from '../user';
+import type {
```

Comment:
Added `type` when importing types as it clarifies that these imports are not part of the compiled JavaScript and do not have a runtime impact.

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export